cpp/src/text/normalize.cu

```diff
@@ -199,8 +200,10 @@ std::unique_ptr<cudf::column> normalize_characters(cudf::strings_column_view con
   if (strings.is_empty()) return cudf::make_empty_column(cudf::data_type{cudf::type_id::STRING});
 
   // create the normalizer and call it
-  data_normalizer normalizer(stream, do_lower_case);
-  auto result = [&strings, &normalizer, stream] {
+  auto result = [&] {
+    auto const cp_metadata = get_codepoint_metadata(stream);
+    auto const aux_table   = get_aux_codepoint_data(stream);
+    data_normalizer normalizer(cp_metadata.data(), aux_table.data(), do_lower_case);
```

Comment:
Since all other lines are using `auto const` style, please also use it for the line above for consistency.